src/test/java/UseCase/PPlaybackTest.java

```diff
@@ -0,0 +1,90 @@
+package UseCase;
+
+import Entities.MusicPlayer;
+import Entities.Song;
+import InputData.playlistInputData;
+import org.junit.Before;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+
+public class PPlaybackTest {
+    @Before
+    public void setUp(){
+    }
+
+    @Test
+    public void testPlay(){
+        File file = new File("./src/test/java/UseCase/test1.mp3");
+        Song song = new Song(0, null, null, 0, null, file, false, null);
+        File file2 = new File("./src/test/java/UseCase/test2.mp3");
```

Comment:
Assuming these are temporary?

Reply:
They can be, but I figured it doesn't really matter since it's the test folder and not the main one.